src/main/java/com/github/princesslana/eriscasper/api/robot/Robot.java

```diff
@@ -68,8 +68,8 @@ public void respond(String regex, Function<RobotContext, Completable> f) {
    * Checks all messages sent directly to the Robot against the provided regex. Executes the
    * provided function when there is a match.
    *
-   * <p>Messages are considered to be directed at the Robot if they begin with "+", the username of
-   * the bot account, or a mention of the bot account.
+   * <p>Messages are considered to be directed at the Robot if they begin with "+", the username on
+   * the bot account, or a mention on the bot account.
```

Comment:
why are all these 'of's replaced with 'on'. I don't think it's correct in all cases.

Reply:
Yeah I don't actually know when I did that.